core/input_map.h

```diff
@@ -76,6 +76,9 @@ class InputMap : public Object {
 
 	const List<Ref<InputEvent> > *get_action_list(const StringName &p_action);
 	bool event_is_action(const Ref<InputEvent> &p_event, const StringName &p_action) const;
```

Comment:
again, add float *r_strength=NULL, bool *r_pressed=NULL as extra optional arguments to event_is_action instead of adding all the functions below.

Reply:
I can't do that, this function is exposed to GDscript which does not handles pointers. 
That's why I added the event_get_action_status.